docs/docs/reference/sdk/search/usePagination.mdx

```diff
@@ -0,0 +1,46 @@
+import PropsComponent from "@site/src/components/PropsComponent/PropsComponent";
+
+# usePagination
+
+The `usePagination` hook provides the pagination links (i.e., `next`/`prev`) of a search result. You can use the `rel="next"` and `rel="prev"` links along with the `<link />` tag to add navigation buttons to your search result pages.
+
+## Import
+
+```tsx
+import { usePagination } from '@faststore/sdk'
+```
+
+## Usage
+
+```tsx
+function Component () {
+  const totalProducts = 100
+  const { next, prev } = usePagination(totalProducts)
+
+  return (
+    <>
+      {prev && <a href={prev.link}>Previous Page</a>}
+      {next && <a href={next.link}>Next Page</a>}
```

Comment:
Recommend adding key props to the example.

Reply:
key? for what?